day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,16 @@
+function speckledFrogs(numberOfFrogs){
+  for ( var i = numberOfFrogs; i > 0; i--) {
+    if (i > 1) {
+      console.log(i + " speckled frogs sat on a log eating some most delicious bugs." +
+      "\n One jumped in the pool where its nice and cool, then there were " + (i - 1)
+      + " speckled frogs.\n\n");
+    }
+    else {
+      console.log(i + " speckled frogs sat on a log eating some most delicious bugs." +
+      "\nOne jumped in the pool where its nice and cool, then there were no more" +
+      " \nspeckled frogs!")
+    }
+  }
+}
+
```

Comment:
Nice work setting this up in a function that can be run for any number of frogs! Don't forget the special circumstance for when i = 2: the language is slightly different there!

Also, I suggest putting special cases in the if/elseif block... I would put the 'default' condition in the last else block. Right now, it seems like the most hit condition is in the first block.